### DISCUSSION TITLE
Is it possible to expand special registers in the config file?

### DISCUSSION THREAD

**Discussion by quantonganh** [helix-editor/helix#7883] [Q&A] [answered]
Hi,

Currently, I'm using #6979 to [run my code](https://github.com/helix-editor/helix/issues/1976#issuecomment-1645104621). But since #6985 has been merged, I would like to give it a try.

To run my code, I can just:

```
:sh run.sh <C-r>%<enter>
```

My question is how can I expand the `%` register in the config file, so I can bind it to a shortcut?

```
[keys.normal.";"]
r = ":sh run.sh <what_to_insert_here_to_expand_%_register?>"
```

Thanks,

**Comment by the-mikedavis** [comment, marked as answer]
That kind of scripting would need https://github.com/helix-editor/helix/issues/1383. Alternatively this could be solved by https://github.com/helix-editor/helix/issues/3134

**Comment by quantonganh** [comment]
> That kind of scripting would need #1383

Thanks for letting me know.

> Alternatively this could be solved by #3134

Actually, I'm using #6979 already.
